src/black.ts

```diff
@@ -0,0 +1,209 @@
+const blackVersions: Fig.Generator = {
+  script: "gh release list --repo psf/black",
```

Comment:
We can add a couple days of cache here, WDYT?
```suggestion
  script: "gh release list --repo psf/black",
  cache: {
    ttl: 1000 * 60 * 60 * 24 * 2, // 2 days
  },
```

Reply:
sure, that makes sense! I've added your suggestions.